### PR TITLE
Completed Mailroom04 Unit Testing

### PR DESCRIPTION
**Pull request by jammychong**

### CODE REVIEW

File: students/jammy_chong/lesson06/David Pokrajac_11252019.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Dear David Pokrajac,
```
Comment (majstorpoki) [inline students/jammy_chong/lesson06/David Pokrajac_11252019.txt:1]: I am not that rich :)

**Review by majstorpoki — COMMENTED** [review]
Thanks, David

File: students/jammy_chong/lesson06/David Pokrajac_11252019.txt (hunk 1 above)
Comment (jammychong) [reply students/jammy_chong/lesson06/David Pokrajac_11252019.txt:1]: Me neither, but we all can dream :)

Comment (majstorpoki) [reply students/jammy_chong/lesson06/David Pokrajac_11252019.txt:1]: :)

**Pull request merged**